pkg/controller/common/rollout/rollout_plan_init.go

```diff
@@ -0,0 +1,19 @@
+package rollout
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/klog/v2"
```

Comment:
I am curious why we choose this lib as it seems not yet widely adopted:)

Reply:
we need to clean up the log system in the whole project.